config/idseq-bench-config.json

```diff
@@ -0,0 +1,55 @@
+{
+  "README": [
+    "********************************************************************************",
+    " This file is under version control at idseq-web/config/idseq-bench-config.json ",
+    " It is consumed by pipeline_monitor.rake.                                       ",
+    "                                                                                ",
+    " You can test and deploy changes to it by running                               ",
+    "                                                                                ",
+    "    aws s3 cp idseq-bench-config.json s3://idseq-web/config.json                ",
+    "                                                                                ",
+    " This process allows changing webapp configuration without the cost             ",
+    " and disruption of a production push.                                           ",
+    "                                                                                ",
+    " The recommended process for deploying a new benchmark is as follows:           ",
+    "                                                                                ",
+    "    1. Branch idseq-bench, adjust to taste, commit all changes.                 ",
+    "                                                                                ",
+    "    2. aws s3 cp --recursive <idseq-bench-output> s3://idseq-bench/<number>     ",
+    "                                                                                ",
+    "    3. Deploy s3://idseq-bench/<number> by prepending to active_benchmarks.     ",
+    "                                                                                ",
+    "    4. After a day or two, move any older entries to retired_benchmarks.        ",
+    "                                                                                ",
+    "********************************************************************************"
+  ],
+  "defaults": {
+    "how_to_use_these_defaults":
+      "These properties may be overridden in each entry of active_benchmarks below.",
+    "project_name": "IDSeq Bench",
+    "user_email": "bdimitrov@chanzuckerberg.com",
```

Comment:
Before the decision to move this config under version control, I was happy to put my user email here.   Now I am less sure.  Any thoughts/concerns?

Reply:
Well, it does open it up to spammers more, but I don't really know if there's an alternative? maybe a special benchmark user?

Reply:
What is the email used for? Seems to me that should be some team e-mail... specially given you're leaving.

Reply:
since i'm not leaving CZI, this email will remain valid... but yeah we probably could create a special account, it's just a matter of config change to switch to it

the benchmarking projects will likely be public